spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/trace/StackdriverTraceAutoConfiguration.java

```diff
@@ -118,8 +132,19 @@ public ManagedChannel stackdriverSenderChannel() {
 				.build();
 	}
 
-	@Bean
-	@ConditionalOnMissingBean
+	@Bean(REPORTER_BEAN_NAME)
+	public Reporter<Span> stackdriverReporter(ReporterMetrics reporterMetrics,
+			GcpTraceProperties trace, @Qualifier(SENDER_BEAN_NAME) Sender sender) {
+		return AsyncReporter.builder(sender).queuedMaxSpans(1000) // historical
+																	// constraint. Note:
+																	// AsyncReporter
+																	// supports memory
+																	// bounds
```

Comment:
Suspicious indents. Isn't it?

Reply:
Let me check what has happened here, I guess it's to auto formatting that changed the comment.